src/parser/deserializer.ts

```diff
@@ -656,17 +643,16 @@ function deserializeObject(
  */
 function isolateEvalWithHash(
```

Comment:
I think its worth renaming this, maybe event merging it with the other one and adding a flag to indicate whether to cache or not.

Reply:
Hey I consolidated the functions here.